src/templates/mkt_nav.html

```diff
@@ -15,6 +24,18 @@
     {{ nav_item(url('new'), _('New'), 'new') }}
     {{ nav_item(url('recommended'), _('Recommended'), 'recommended') }}
     {{ nav_item('#', _('Categories'), 'categories', 'categories') }}
+    <div class="mkt-nav-root--divider"></div>
```

Comment:
Seems like it should really be more like:

```html
<ul>
  <li ...><a ...>Home
  <li ...><a ...>Popular
</ul>
<ul class="mkt-nav--settings">
  <li ...><a ...>Feedback
</ul>
```

It would be more semantic and you could then style `.mkt-nav--settings:before` to set the divider.